core/src/main/java/com/adobe/guides/aem/components/core/models/CategoryListImpl.java

```diff
@@ -84,7 +84,7 @@ private void initModel() {
 
         try {
             templateName = getTemplateName(cqTemplate);
-            categoryList = findPagesByTemplate(request.getResourceResolver(), FilenameUtils.concat(SLASH, FilenameUtils.concat(CONF_PATH_TOKEN,FilenameUtils.concat(templateName , TEMPLATE_PATH_TOKEN))), FilenameUtils.concat(SLASH, FilenameUtils.concat(CONTENT_PATH_TOKEN, templateName)));
+            categoryList = findPagesByTemplate(request.getResourceResolver(), FilenameUtils.concat(SLASH, FilenameUtils.concat(CONF_PATH_TOKEN,FilenameUtils.concat(templateName , TEMPLATE_PATH_TOKEN))), currentPage.getPath());
```

Comment:
Earlier we were searching from "/content/template" and now we are searching at "/content/template/docs/en" . Am I correct here? currentPage.getPath() =/content/template/docs/en"

Reply:
To list all the categories under "/content/<site_name>/en", shouldn't we provide this as path, because currentPage.getPath() will give "/content/<site_name>/en/docs".

Reply:
wherever the component is used, it will list its children pages of the product page type.